x-pack/filebeat/tests/integration/input_reload_test.go

```diff
@@ -0,0 +1,336 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/elastic/beats/v7/x-pack/libbeat/management"
+	"github.com/elastic/elastic-agent-client/v7/pkg/client/mock"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+)
+
+// TestInputReloadUnderElasticAgent will start a Filebeat and cause the input
+// reload issue described on https://github.com/elastic/beats/issues/33653
```

Comment:
I would prefer to have a short issue description here in the comments instead of a link to the Github issue.

Reply:
Done on efd1811910